os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -1019,7 +1019,9 @@ u32 app_mpu_nocache_init(void)
 
 u32 app_mpu_s_nocache_init(void)
 {
+#ifdef CONFIG_AMEBAD_TRUSTZONE
```

Comment:
you can put this condition at app_mpu_s_nocache_init call itself?

Reply:
The reason to put the condition here is to prevent compile error went TrustZone is not enable.
mpu_s_no_cache_init() API only exist image3, the Secure lib/image3.
If put this condition at app_mpu_s_nocache_init call itself compile error will still happen.
Unless I put the condition in both location.